lib/PuppeteerSharp/Connection.cs

```diff
@@ -276,14 +276,27 @@ private void ProcessResponse(string response)
             }
         }
         #endregion
+
         #region Static Methods
 
-        internal static async Task<Connection> Create(string url, int delay = 0, int keepAliveInterval = 60, ILoggerFactory loggerFactory = null)
+        /// <summary>
+        /// Gets default <see cref="WebSocketFactory"/> implementation.
+        /// </summary>
+        public static readonly WebSocketFactory DefaultWebSocketFactory = async (uri, options, cancellationToken) =>
+        {
+            var result = new System.Net.WebSockets.ClientWebSocket();
+#pragma warning disable 618
+            result.Options.KeepAliveInterval = TimeSpan.FromMilliseconds(options.KeepAliveInterval);
+#pragma warning restore 618
```

Comment:
Why are we doing this?

Reply:
KeepAliveInterval is marked obsolete

Reply:
I don't see it marked as obsolete https://github.com/dotnet/corefx/blob/master/src/System.Net.WebSockets.Client/src/System/Net/WebSockets/ClientWebSocketOptions.cs#L155

Do you have any doc?